tasty-query/shared/src/main/scala/tastyquery/reader/Loaders.scala

```diff
@@ -18,13 +18,12 @@ import tastyquery.reader.tasties.TastyUnpickler
 
 private[tastyquery] object Loaders {
 
-  class MissingTopLevelTasty(root: Loader.Root) extends Exception(s"Missing TASTy for ${root.fullName}")
-  class UnexpectedTasty(root: Loader.Root) extends Exception(s"Unexpected TASTy for ${root.fullName}")
+  class MissingTopLevelTasty(root: Loader.Root) extends Exception(s"Missing TASTy for $root")
+  class UnexpectedTasty(root: Loader.Root) extends Exception(s"Unexpected TASTy for $root")
 
   object Loader:
     private[Loaders] case class Root private[Loaders] (pkg: PackageSymbol, rootName: SimpleName):
-      def fullName: FullyQualifiedName =
-        pkg.fullName.select(rootName)
+      override def toString(): String = pkg.displayFullName + "." + rootName.toString
```

Comment:
this will now print `<empty>.Foo` rather than `Foo`

Reply:
Fixed.